build/package/launchd/Library/LaunchDaemons/com.newrelic.infrastructure.plist

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>Label</key>
+        <string>com.newrelic.infrastructure</string>
+
+        <key>Program</key>
+        <!-- TODO replace with final path -->
+        <string>/tmp/newrelic-infra/newrelic-infra-service</string>
+
+        <key>WorkingDirectory</key>
+        <!-- TODO replace with final path -->
+        <string>/tmp/newrelic-infra/</string>
+
+        <key>RunAtLoad</key>
+        <true/>
+
+        <!-- KeepAlive prevents stop via: launchctl stop com.newrelic.infrastructure
+        <key>KeepAlive</key>
+        <true/>
+        <key>ThrottleInterval</key>
+        <integer>30</integer>
+        -->
+
+        <!--
+        <key>EnvironmentVariables</key>
+        <dict></dict>
+        -->
+
+        <key>StandardOutPath</key>
+        <string>/var/log/newrelic-infra.log</string>
```

Comment:
I think we will need to review all the paths after deciding if we are running the agent as root or not. ☝️